src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,7 +25,10 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetEventSource is only part of .NET Core")]
+        [SkipOnTargetFramework(
+            TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.Uap, 
```

Comment:
You might not need to specify both `TargetFrameworkMonikers.UapAot` and `TargetFrameworkMonikers.Uap`.  

`TargetFrameworkMonikers.Uap` was supposed to include both uap and uap-aot configurations. But @danmosemsft knows whether that fix made it into the branch now.